admin/client/App/screens/Item/index.js

```diff
@@ -79,19 +99,25 @@ var ItemView = React.createClass({
 			<div className="Relationships">
 				<Container>
 					<h2>Relationships</h2>
-					{keys.map(key => {
-						const relationship = relationships[key];
-						const refList = listsByKey[relationship.ref];
-						return (
-							<RelatedItemsList
-								key={relationship.path}
-								list={this.props.currentList}
-								refList={refList}
-								relatedItemId={this.props.params.itemId}
-								relationship={relationship}
-							/>
-						);
-					})}
+					<div className="ItemList-wrapper">
+						<table cellPadding="0" cellSpacing="0" className="Table ItemList">
+							<colgroup>{[this.renderTableCols()]}</colgroup>
+							{this.renderRelationshipTableHeaders()}
+							{keys.map(key => {
+								const relationship = relationships[key];
+								const refList = listsByKey[relationship.ref];
+								return (
+									<RelatedItemsList
+										key={relationship.path}
+										list={this.props.currentList}
+										refList={refList}
+										relatedItemId={this.props.params.itemId}
```

Comment:
you could put this and this.props.currentList in a const at the top of the function to increase clarity and reduce repeated deferencing in the loop.